src/RoslynInsertionTool/RoslynInsertionTool/RoslynInsertionTool.VisualStudioTeamServices.cs

```diff
@@ -359,9 +359,11 @@ internal static async Task<InsertionArtifacts> GetInsertionArtifactsAsync(Build
             {
                 if (artifact.Name == arcadeArtifactName)
                 {
+                    var artifactType = artifact.Resource.Type;
                     // artifact.Resource.Data should be available and non-null due to BuildWithValidArtifactsAsync,
                     // which checks this precondition
-                    if (!StringComparer.OrdinalIgnoreCase.Equals(artifact.Resource.Type, "container"))
+                    if (!artifactType.Equals("container", StringComparison.OrdinalIgnoreCase) &&
+                        !artifactType.Equals("pipelineArtifact", StringComparison.OrdinalIgnoreCase))
```

Comment:
I guess this matches this field https://github.com/dotnet/roslyn/blob/cc6e5bd358cbf9fccde8f017446d9843744f058f/azure-pipelines-official.yml#L164

Reply:
wondering - why are our insertions not currently erroring?

Reply:
I belive if you run RIT from the signed build pipeline, it find the VSSetup folder locally.
But when running it offline, it will try to download it

Reply:
I think I experienced this same issue when running locally, which I dealt with by downloading the artifact manually and passing the path to it.

Reply:
So should we consider the change here is OK now? (it's not a blocking issue for our daily insertion)

Reply:
I think we should take your PR, I'll wrap up reviewing it shortly